casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -73,15 +73,7 @@ class RuntimeManager private (val emptyStateHash: ByteString, runtimeContainer:
     result
   }
 
-  def computeBonds(hash: StateHash)(implicit scheduler: Scheduler): Seq[Bond] =
```

Comment:
So, previous implementation was catching the exception and printing it (on the server side) but now you allow it to bubble all the way up to `DeployGrpcService` ?